website/app-templates/smarty/help-pages/en/help.html

```diff
@@ -748,10 +723,10 @@ <h2>Reference Number</h2>
                     <div class="caption">
                         <h3>e.g. GK003F</h3>
                         <p>
-                            It is a <u>public</u> number that may be used to refer a GeoKret.
-                            It consists of 'GK' prefix and 4 digits in hexadecimal system.
-                            You can use it to search a specific GeoKret or to link to
-                            this GeoKret. It is visible to everyone.
+                            This is a <u>public</u> number that may be used to refer a GeoKret.
+                            It consists of the 'GK' prefix and 4 digits in the hexadecimal system.
```

Comment:
Time passed our numbers are 4 to 5 chars

Reply:
Changed this!